### PR TITLE
refactor: minor changes after review

### PR DESCRIPTION
**Pull request by crazydevlegend**

### CODE REVIEW

**Comment by crazydevlegend** [comment]
@afterpartyai i'll be pushing more commits to refactor the code
btw, i've noticed some places where you wrote/added for the testing purposes - i believe, we need to tidy up the code to make it production-ready

File: conversationgenome/utils/config.py [hunk 1]
```diff
@@ -151,14 +151,14 @@ def add_miner_args(cls, parser):
...
-        default=False,
+        default=True,
     )
 
     parser.add_argument(
         "--blacklist.allow_non_registered",
         action="store_true",
         help="If set, miners will accept queries from non registered entities. (Dangerous!)",
-        default=False,
+        default=True,
```
Comment (afterpartyai) [inline conversationgenome/utils/config.py:161]: I believe this should be set to false for security. Can you explain why we would want this to default to True?

Comment (afterpartyai) [reply conversationgenome/utils/config.py:161]: I'm going to change this one to False and merge your PR. If you think it should be true, we can discuss it and do another PR.

Comment (crazydevlegend) [reply conversationgenome/utils/config.py:161]: oh, was my mistake, thought it was `blacklist_non_registered`

**Pull request merged**